store/postgres/src/deployment.rs

```diff
@@ -890,33 +893,24 @@ pub fn create_deployment(
                 features,
                 schema,
             },
-        failed,
-        health: _,
-        synced,
-        fatal_error: _,
-        non_fatal_errors: _,
         earliest_block,
-        latest_block,
         graft_base,
         graft_block,
         debug_fork,
-        reorg_count: _,
-        current_reorg_depth: _,
-        max_reorg_depth: _,
     } = deployment;
 
     let deployment_values = (
         d::id.eq(site.id),
         d::deployment.eq(site.deployment.as_str()),
-        d::failed.eq(failed),
-        d::synced.eq(synced),
+        d::failed.eq(false),
+        d::synced.eq(false),
         d::health.eq(SubgraphHealth::Healthy),
         d::fatal_error.eq::<Option<String>>(None),
         d::non_fatal_errors.eq::<Vec<String>>(vec![]),
         d::earliest_ethereum_block_hash.eq(b(&earliest_block)),
         d::earliest_ethereum_block_number.eq(n(&earliest_block)),
-        d::latest_ethereum_block_hash.eq(b(&latest_block)),
-        d::latest_ethereum_block_number.eq(n(&latest_block)),
+        d::latest_ethereum_block_hash.eq(sql("null")),
+        d::latest_ethereum_block_number.eq(sql("null")),
```

Comment:
Not critical, but I think this could be expressed as `None`, since those fields are `Nullable`. 
https://github.com/graphprotocol/graph-node/blob/f267b3d2aa6908ce024a94244378b4e91c9f21f5/store/postgres/src/deployment.rs#L69-L70
```suggestion
        d::latest_ethereum_block_hash.eq(None),
        d::latest_ethereum_block_number.eq(None),
```

Reply:
Yeah, that's what I tried to do first, but you have to have pretty ugly type annotations to explain the `Option<_>` type to the compiler, so I copped out and just used `null`.